cron/rotateIndexes.py

```diff
@@ -65,9 +66,11 @@ def esRotateIndexes():
             options.dobackup, options.rotation, options.pruning):
             try:
                 if rotation != 'none':
+                    previndex = index
                     oldindex = index
```

Comment:
My guess is that we are going to lose the thread on the distinction between previndex & oldindex after a while. We should either comment clearly or refactor to have todayindex, yesterdayindex, twodaysagoindex

Reply:
Definitely agreed, I'll add comments in a little bit.

Reply:
Done